guides/app-guides/leverage.md

```diff
@@ -0,0 +1,49 @@
+---
+description: How to get leverage on crypto by borrowing agEUR in Angle
+---
+
+# Leverage on crypto with agEUR
+
+The Angle Borrowing module can be used to get leverage on crypto deposited as collateral with agEUR, directly from the [Angle App](https://app.angle.money/#/borrow). 
+
+This works by borrowing agEUR, swapping them into the token originally deposited as collateral, and depositing this token back into the vault. You can learn more about the mechanism [here](/borrowing-module/vaults/README.md#leveraging-collateral-exposure). 
+
+Note that at the difference of simple borrowing, leveraging increases your exposure to the collateral token and make the overall position more risky.  
+
+## How to leverage
+
+### Steps
+
+Here are the steps to get leverage with Angle: 
+1. Select your vault and the `Leverage` action. 
+2. Input the amount of collateral that you want to deposit
+3. Input the amount of additional exposure you want on the collateral token, or choose your leverage from the slider.
+4. Make sure to double check the summary on the screen, and click on `Add step` to move on to the final section and finalize the transaction. 
```

Comment:
Add step 5: Approve/Sign permit
Optional Step 6: Sign permit for router
Step 7: Send tx